src/elements/content-sidebar/activity-feed/task-form/__tests__/TaskForm.test.js

```diff
@@ -439,36 +440,46 @@ describe('components/ContentSidebar/ActivityFeed/task-form/TaskForm', () => {
             );
         });
 
-        test('should not show completion rule checkbox when feature is off', () => {
-            const approvers = new Array(3).fill().map(() => ({
-                id: '',
-                target: {
-                    id: 123 * Math.random(),
-                    name: 'abc',
-                    type: 'user',
-                },
-                role: 'ASSIGNEE',
-                type: 'task_collaborator',
-                status: 'NOT_STARTED',
-                permissions: { can_delete: false, can_update: false },
-            }));
-            const wrapper = renderWithFeatures(
-                {
```

Comment:
in the previous code, the parameters for this helper function were transposed, so this test always passed. this fixes that.